### PR TITLE
Add Timescale Cloud security doc

### PR DESCRIPTION
**Pull request by CoolAssPuppy**
This is part of our blog tutorial >> docs migration process. Question for this PR: should we also expand this page to include more information about securing Timescale Cloud that's currently locked away in the knowledgebase? (KB link is at the end of the doc)

Original blog post: https://blog.timescale.com/tutorials/how-to-secure-network-access-to-your-timescale-cloud-service/

### CODE REVIEW

File: getting-started/securing-timescale-cloud.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+# How to secure network access to your Timescale Cloud service
+
+It's the easiest question to ask, but the hardest to answer about any system: 
+**is it secure?** Having a secure system doesn't have a binary "yes or no" answer 
+like the question implies. Security is implemented in layers and each choice 
+comes with tradeoffs. And in a world where you are using cloud services with your 
+system, this topic becomes even more important and complex.
```
Comment (solugebefola) [inline getting-started/securing-timescale-cloud.md]: The tone here seems not-quite-docs-like.  Maybe better to start with the next paragraph, but change the opening to something like
> One very critical piece of securing your database within Timescale Cloud is network protection.

etc etc

File: getting-started/securing-timescale-cloud.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# How to secure network access to your Timescale Cloud service
```
Comment (solugebefola) [inline getting-started/securing-timescale-cloud.md]: Title is a bit long:

Screen Shot 2020-02-29 at 11 05 30 PM

File: getting-started/securing-timescale-cloud.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+get signed up and create your first database instance.
+
+### Step 1 - Navigate to your TimescaleDB instance
+
+Once you have a database instance setup in the [Timescale Cloud portal][timescale-portal],
+browse to this service and click on the “Overview” tab. In the “Connection Information” 
+section, you will see the port number that is used for database connections. This is 
+the port we will protect by managing inbound access.
+
+<img class="main-content__illustration" src="https://s3.amazonaws.com/docs.timescale.com/securing-timescale-cloud/overview-tab.png" alt="Timescale Cloud Oerview tab"/>
```
Comment (solugebefola) [inline getting-started/securing-timescale-cloud.md]:
```suggestion
<img class="main-content__illustration" src="https://s3.amazonaws.com/docs.timescale.com/securing-timescale-cloud/overview-tab.png" alt="Timescale Cloud Overview tab"/>
```

Comment (solugebefola) [reply getting-started/securing-timescale-cloud.md]: I assume this image will be put in at the src location soon?  It's not there now.

File: getting-started/securing-timescale-cloud.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+### Step 2 - Find the allowed IP addresses section
+
+Scroll down to find the "Allowed IP Addresses" section. By default, this value is set to 
+`0.0.0.0/0` which is actually wide-open.
+
+>:WARNING: This wide-open setting simplifies getting started since it will accept incoming traffic from all sources, but you will absolutely want to narrow this range.
+
+If you are curious how to interpret this [Classless Inter-Domain Routing][cidr-wiki] (CIDR) syntax,
```
Comment (solugebefola) [inline getting-started/securing-timescale-cloud.md]:
```suggestion
If you are curious about how to interpret this [Classless Inter-Domain Routing][cidr-wiki] (CIDR) syntax, 
```

File: getting-started/securing-timescale-cloud.md (hunk 3 above)
Comment (CoolAssPuppy) [reply getting-started/securing-timescale-cloud.md]: Screenshots are in Dropbox: https://www.dropbox.com/sh/1s5039hht5qumxx/AAB5uGFgVGjqjgbJRZJsGwZaa?dl=0

**Comment by solugebefola** [comment]
Please label with version

**Pull request merged**